docs/xonshconfig.rst

```diff
@@ -42,27 +42,39 @@ dictionaries have the following structure:
     ``default="env"``
 :aliascmd: *str, optional* - The command to generate alais output with.
     ``default="alias"``
-:extra_args: *list of str, optional* - Addtional command line options to pass 
+:extra_args: *list of str, optional* - Addtional command line options to pass
     into the shell. ``default=[]``
 :currenv: *dict or null, optional* - Manual override for the current environment.
     ``default=null``
-:safe: *bool, optional* - Flag for whether or not to safely handle exceptions 
+:safe: *bool, optional* - Flag for whether or not to safely handle exceptions
     and other errors. ``default=true``
-:prevcmd: *str, optional* - An additional command or script to run before 
-    anything else, useful for sourcing and other commands that may require 
+:prevcmd: *str, optional* - An additional command or script to run before
+    anything else, useful for sourcing and other commands that may require
     environment recovery. ``default=''``
 :postcmd: *str, optional* - A command to run after everything else, useful for
     cleaning up any damage that the ``prevcmd`` may have caused. ``default=''``
-:funcscmd: *str or None, optional* - This is a command or script that can be 
+:funcscmd: *str or None, optional* - This is a command or script that can be
     used to determine the names and locations of any functions that are native
-    to the foreign shell. This command should print *only* a whitespace 
+    to the foreign shell. This command should print *only* a whitespace
     separated sequence of pairs function name & filenames where the functions
     are defined. If this is None (null), then a default script will attempted
-    to be looked up based on the shell name. Callable wrappers for these 
+    to be looked up based on the shell name. Callable wrappers for these
     functions will be returned in the aliases dictionary. ``default=null``
-:sourcer: *str or None, optional* - How to source a foreign shell file for 
-    purposes of calling functions in that shell. If this is None, a default 
+:sourcer: *str or None, optional* - How to source a foreign shell file for
+    purposes of calling functions in that shell. If this is None, a default
     value will attempt to be looked up based on the shell name. ``default=null``
+:runcmd : *str or None, optional* - Command line switches to use when
+    running the script, such as ``-c`` for Bash and ``/C`` for cmd.exe.
+    ``default=null``
+:seterrprevcmd : *str or None, optional* - Command that enables exit-on-error
+    for the shell before all other commands. For example, this is "set -e" in Bash.
+    To disable this exit-on-error behavior, simply pass in an empty string.
+    ``default=null``
+:seterrpostcmd : *str or None, optional* - Command that enables exit-on-error
+    for the shell after all other commands. For example, this is "set -e" in Bash.
+    To disable this exit-on-error behavior, simply pass in an empty string.
```

Comment:
Does it make sence to use `set -e` for seterrpostcmd ? If not just remove the 'For example, ...'